src/_locales/tel.config

```diff
@@ -0,0 +1,186 @@
+@extension_description
+ప్రతి వెబ్‌సైట్‌కి ముదురు థీమ్. మీ కళ్లను కాపాడుకోండి, రాత్రి మరియు పగలు బ్రౌజింగ్ట్‌కి ముదురు థీమ్ ఉపయోగించండి. 
+
+@loading_please_wait
+దయచేసి వేచి ఉండండి...
+
+
+#====Top section ====
+
+@on
+ఆన్ 
+
+@off
+ఆపండి
+
+@toggle_current_site
+సైట్ కోసం సక్రియం చేయండి
+
+@setup_hotkey_toggle_site
+సైట్ మార్చడానికి పొడిగింపు సెటప్
+హాట్‌కీని ప్రారంభించండి
+
+@toggle_extension
+పొడిగింపును ప్రారంభించండి
+
+@setup_hotkey_toggle_extension
+పొడిగింపు సెట్టింగులు
+హాట్‌కీని ప్రారంభించండి
+
+@automation
+స్వయంచాలకంగా పూర్తయింది
+
+@set_active_hours
+మేల్కొనే గంటలను మార్చండి
+
+@set_location
+సూర్యాస్తమయం వద్ద ప్రారంభించండి
+
+@longitude
+రేఖాంశం
+
+@latitude
+అక్షాంశం
+
+@page_protected
+ఈ పేజీ బ్రౌజర్ ద్వారా రక్షించబడింది
```

Comment:
Please use two sentences for this translation.

Reply:
@page_protected
ఈ పేజీ
బ్రౌజర్ ద్వారా రక్షించబడింది

Reply:
I have split the translation into two lines like the translations in other languages. I have done this for @page_protected and @page_in_dark_list.

Reply:
Thank you for the feedback